barman/backup_executor.py

```diff
@@ -562,7 +565,7 @@ def fetch_remote_status(self):
 
         return remote_status
 
-    def backup_copy(self, backup_info):
+    def backup_copy(self, backup_info, parent_backup_info=None):
```

Comment:
All the signature changes have to happen in the base abstract class...  otherwise, we are not correctly implementing the abstract methods (and just overloading on the methods names)